<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(94, 92, 100);</string>
   </property>
   <widget class="QLineEdit" name="InputField">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgb(255, 255, 255);
border-radius: 5%;
color: rgb(222, 221, 218);
padding: 2%;
</string>
    </property>
    <property name="placeholderText">
     <string>Enter how many six-sided dice you want to roll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(87, 227, 137);
border-radius: 10%;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>129</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 221, 218);
border: 1px solid #fff;
border-radius: 5%; </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
